.github/workflows/deploy.yml

```diff
@@ -0,0 +1,47 @@
+name: Push new build
+on:
+  pull_request: null
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - uses: actions/cache@v3
+        with:
+          path: Library
+          key: Library-${{ hashFiles('Assets/**', 'Packages/**', 'ProjectSettings/**') }}
+          restore-keys: |
+            Library-
+      - name: Build Project
+        uses: game-ci/unity-builder@v4
+        env:
+            UNITY_LICENSE: ${{ secrets.UNITY_LICENSE }}
+            UNITY_EMAIL: ${{ secrets.UNITY_EMAIL }}
+            UNITY_PASSWORD: ${{ secrets.UNITY_PASSWORD }}
+        with:
+            targetPlatform: WebGL
+      
+      - name: Set version info
+        run: |
+          RELEASE_DATE=$(date --rfc-3339=date)
+          SHORT_HASH=$(git rev-parse --short HEAD)
+          echo "VERSION=${RELEASE_DATE}-${SHORT_HASH}">> ${GITHUB_ENV}
+
+      - name: Upload build to itch.io
+        uses: robpc/itchio-upload-action@v1
+        with:
+          path: build
+          project: mckornfield/alien-arena # TODO: Replace with a service address
```

Comment:
TODO like future TODO? If so please create a ticket 👌  first CH&M ticket baybeeeeeeee

Reply:
🍾

Reply:
oh noooo

Reply:
created https://trello.com/c/LALhJQuY/86-use-a-company-itchio-and-unity-account-instead-of-mckornfield